src/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  width: 1024px;
+
+  &__card {
+    margin: 0 48px 46px 0;
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
```

Comment:
let's apply `:not()`

Reply:
why? it's just alternative way. Or consequential?